testing/benchmark/README.md

```diff
@@ -101,9 +101,3 @@ do that in `terraform.tfvars.example`.
 
 `make cleanup-elasticsearch` will delete all the APM data streams. This may be useful in case you'd like to re-run
 the benchmarks without destroying the deployment.
-
-### Slack reporting
-Reporting data is taken from the [app_bench_diff_shifts_slack](https://observability-benchmarks.elastic.dev/app/dashboards#/view/a5bc8390-2f8e-11ed-a369-052d8245fa04?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-30d,to:now)))
```

Comment:
if the docs are relevant but the URL is not required to be public available, what do you think replacing the FQDN for <replace-with-kibana-benchmark-url>`?